src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1611,8 +1723,7 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     return UCS_OK;
 
 err_destroy_fc_ep_and_dcis:
-    uct_dc_mlx5_iface_dcis_destroy(self, self->tx.num_dci_pools,
-                                   uct_dc_mlx5_iface_total_ndci(self));
+    uct_dc_mlx5_iface_dcis_destroy(self, self->tx.num_dci_pools);
```

Comment:
maybe remove 2nd arg for uct_dc_mlx5_iface_dcis_destroy? seems it's always self->tx.num_dci_pools so the function can take it from the iface pointer